src/av2/structures/ndgrid.py

```diff
@@ -121,13 +121,16 @@ class BEVGrid(NDGrid):
     """
 
     def points_to_bev_img(
-        self, points: NDArrayFloat, color: Tuple[int, int, int] = GRAY_BGR, diameter: int = 2
+        self,
+        points: NDArrayFloat,
+        color: Union[Tuple[int, int, int], NDArrayByte] = GRAY_BGR,
+        diameter: int = 2,
     ) -> NDArrayByte:
         """Convert a set of points in Cartesian space to a bird's-eye-view image.
 
         Args:
             points: (N,D) List of points in R^D.
-            color: RGB color.
+            color: RGB color or list of grayscale values (0-255) representing lidar intensity.
```

Comment:
```suggestion
            color: RGB color or list of grayscale values [0-255] representing lidar intensity.
```

Could we represent this as a closed interval?

Reply:
Sure!